alchemy-web/docs/providers/vercel/project.md

```diff
@@ -0,0 +1,60 @@
+# Project
+
+Create and manage Vercel projects.
+
+## Authentication
+
+To use this resource, set `VERCEL_ACCESS_TOKEN` in your `.env` file or pass `accessToken` directly in your resource configuration.
+
+## Examples
+
+### With `accessToken`
+
+```ts
+const project = await Project("my-app", {
+  name: "my-app",
+  framework: "astro",
+  accessToken: alchemy.secret(process.env.VERCEL_ACCESS_TOKEN)
+});
+```
+
+### With GitHub
+
+```typescript
+const project = await Project("my-app", {
+  name: "my-app",
+  framework: "nextjs",
+  gitRepository: {
+    type: "github",
+    repo: "username/my-app"
+  },
+});
+```
+
+### With Secrets (Environment Variables)
+
+```ts
+const project = await Project("my-app", {
+  name: "my-app",
+  environmentVariables: [
+    {
+      key: "DATABASE_URL",
+      value: alchemy.secret("DATABASE_URL"),
+      type: "encrypted",
```

Comment:
`type: "encrypted"` could be inferred from `alchemy.secret` like we do for cloudflar `secret_text` vs `text` binding

Reply:
They have 2 versions – `encrypted` and `sensitive` (which can't be decrypted).  I figure that users should decide between the two.

**However**, perhaps your point means that this can be _inferred_ (defaulted to `encrypted`) and a user can choose to make this `sensitive`.

Reply:
Oooo I _like_ this!

Resolved via cffa6b4150b853ecf60f38ca224ead0a8bc996c5